src/js/components/Meter/Circle.js

```diff
@@ -13,12 +13,17 @@ const Circle = forwardRef((props, ref) => {
   const theme = useContext(ThemeContext);
   const width =
     size === 'full' ? 288 : parseMetricToNum(theme.global.size[size] || size);
-  const height =
-    type === 'pie'
-      ? width / 2
-      : parseMetricToNum(theme.global.edgeSize[thickness] || thickness);
+  let height;
+  if (type === 'pie') {
+    height = width / 2;
+  } else if (type === 'semicircle') {
+    height =
+      2 * parseMetricToNum(theme.global.edgeSize[thickness] || thickness);
```

Comment:
Why does this need to be different for "semicircle"? I was expecting the strokeWidth to be the same as for "circle" and "bar".